src/content/docs/tutorial-create-alerts/manage-alert-quality.mdx

```diff
@@ -0,0 +1,182 @@
+```
+  </Collapser>
+
+  <Collapser
+    id="kpi-mttc"
+    title="Mean time to close (MTTC) KPI"
+  >
+This is simply the average duration of incidents within the period of time measured. You want this number to be as low as possible.
+
+**Goal:** Reduce MTTC
+
+**Best practices:**
+* Don't manually close incidents. Manual closure will skew the real duration of incident length.
+* Improve reliability engineering skills.
+* Report AQM KPIs to all stakeholders.
+
+```sql
+FROM NrAiIncident SELECT average(durationSeconds/60) AS 'Incident MTTC (minutes)' WHERE event = 'close' AND priority = 'critical' SINCE 1 WEEK AGO COMPARE WITH 1 WEEK AGO
+```
+  </Collapser>
+
+  <Collapser
+    id="kpi-pct-under-five"
+    title="Percent under 5 minutes KPI"
+  >
+This is the percentage of incidents whose total duration is under five minutes. This can be an indicator of of an incident changing state too frequently, which can obscure the cause and severity of the incident. This state is known as **incident flapping**.
+
+**Goal:** Minimize percentage of incidents with short durations.
+
+**Best practices:**
+* Ensure that conditions are detecting legitimate deviations from expected behavior.
+* Understand service level management.
+* Ensure that conditions are detecting legitimate deviations that correlate to business impact or impending business impact.
+
+```sql
+FROM NrAiIncident SELECT percentage(count(*), WHERE durationSeconds <= 5*60) AS '% Under 5min' WHERE event = 'close' AND priority = 'critical' SINCE 1 WEEK AGO COMPARE WITH 1 WEEK AGO 
+```
+</Collapser>
+</CollapserGroup>
+
+### User engagement [#user]
+
+You should measure the value of an incident by the amount of attention it receives. The amount of engagement an individual alert receives is a direct measurement of its value. More engagement implies a valuable alert, while less (or zero) engagement implies an alert may simply be noisy and should be modified or disabled.
+
+There's a significant difference between measuring the moment of incident awareness and acknowledging when resolution activity begins. If you're using an integration with New Relic alerts, be sure that the `Acknowledge` event sent to New Relic triggers when resolution activity begins, not when the incident is sent to the external incident management tool. 
+
+<Callout variant="tip">
+For more information regarding standard incident management processes, see "[Incident management process: 5 steps to effective resolution, posted on August 31, 2020 by OnPage Corporation.](https://www.onpage.com/incident-management-process-5-steps-to-effective-resolution) in reference to [ITIL4](https://itsm.tools/its-here-itil-4-explained)"
+</Callout>
+
+<CollapserGroup>
+  <Collapser
+    id="kpi-pct-ack"
+    title="Percentage acknowledged KPI"
+  >
+This identifies the percentage of incidents that you've engaged with by having their acknowledgement flag set to `true`. Typically you should compare the current and previous weeks.
```

Comment:
`that you've engaged with by having their acknowledgement flag set to `

a lot of preposition and verbs all fighting here: `that you have.. with by having... set to`